controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -70,6 +70,10 @@ def setUp(self):
         )
 
         self.array = DS8KArrayMediator("user", "password", self.endpoint)
+        volume_cache_by_address_patcher = patch.dict(
+            "controller.array_action.ds8k_volume_cache.volume_cache_by_address._volume_cache_by_address", clear=True)
```

Comment:
I think we can have more abstraction in the unit tests by patching the `VolumeCache` to return a mock, similar to how the ds8k client mediator member is mocked

Reply:
`VolumeCache` does not represent in any way the cache, only refer to it, mocking it wont mock the cache...

Reply:
ok, done